tests/backends/test_utils.py

```diff
@@ -45,3 +47,26 @@ def equal(value, max_d, places, result):
             equal('0.1234567890', 5, None, '0.12346')
         with self.assertRaises(Rounded):
             equal('1234567890.1234', 5, None, '1234600000')
+
+
+class CallProcTest(unittest.TestCase):
```

Comment:
Generally I think we want to use `TestCase` for any test that accesses the database, especially in light of the possibility of https://code.djangoproject.com/ticket/28478.